### ISSUE TITLE
App not updating with latest messages

### ISSUE THREAD

**Issue by niftynei** [zulip/zulip-android#226]
A few users have reported that the newest messages are failing to load.  Killing the app and restarting seems to fix.

**Comment by Sam1301** [comment]
@niftynei  I tested this in the beta and release candidate and it seemed to work. The newest messages are added at the bottom of homeView, though the users have to scroll down to them (without refreshing).

**Comment by niftynei** [comment]
yes, it seems to work in most cases, but there appears to be a hard to replicate state where messages are no longer refreshed.  i managed to get my app into this state, but i don't know how to reproduce reliably.

**Comment by bengoldstein** [comment]
I'm using the Play Store edition release March 1 , 2017, and having this problem. By "this problem" I mean that the posts I make in a topic frequently do not show up unless I refresh. (I do see what other users post. Also, I do see what I post from my Android in my desktop client.)

It *may* have something to do with my using both the desktop client and the Android app at the same time. I'm not able to reliably reproduce this problem either, although it is happening a lot for me--basically making the app unusable. (I have tried this on two cell phones--a Samsung Galaxy S5 and a Google Nexus 5X with the same results.)
